.github/workflows/release-fleet.yml

```
@@ -91,19 +91,22 @@ jobs:
           repo: "ghcr.io/rancher/fleet"
           tag: "${GITHUB_REF_NAME}"
         run: |
+          # Replace rancher/fleet, rancher/fleet-agent and rancher/gitjob image names, but not eg. rancher/kubectl
           sed -i \
-              -e "s@repository:.*@repository: $repo@" \
-              -e "s/tag:.*/tag: ${VERSION}/" \
+              -e "s@repository: rancher/\(\(fleet.*\|gitjob\)\).*@repository: ${repo/fleet/\\1}@" \
+              -e "s/tag:.*/tag: ${tag}/" \
               charts/fleet/values.yaml
 
           sed -i \
-              -e "s@repository:.*@repository: $repo@" \
-              -e "s/tag: dev/tag: ${VERSION}/" \
+              -e "s@repository: rancher/\(\(fleet.*\|gitjob\)\).*@repository: ${repo/fleet/\\1}@" \
+              -e "s/tag: dev/tag: ${tag}/" \
```

`${}` looks important, but is useless since / can't be part of the variable name.
So, I'd suggest:
```suggestion
              -e "s/tag: dev/tag: $tag/" \
```

To be honest, I find shell variables in regex confusing. This isn't much more readable, though:
```
              -e 's/tag: dev/tag: '"$tag"'/' \
```

---

It is useless, apart from emphasizing the presence of a shell variable :upside_down_face: